cmd/atlas/internal/sqlparse/sqliteparse/sqliteparse.go

```diff
@@ -214,8 +250,14 @@ func isnull(t antlr.Tree) bool {
 }
 
 func unquote(s string) string {
-	if len(s) < 2 || s[0] != '`' || s[len(s)-1] != '`' {
-		return s
+	switch {
+	case len(s) < 2:
+	case s[0] == '`' && s[len(s)-1] == '`':
+		return s[1 : len(s)-1]
```

Comment:
I think strconv.Unquote works on backticks as well.

Reply:
Thanks for this. I'll fix it.

Reply:
"`" cannot be escaped, so calling Unquote is unnecessary. But better merging these two together.